openmetadata-ui/src/main/resources/ui/src/components/common/CustomPropertyTable/PropertyValue.tsx

```diff
@@ -87,41 +107,62 @@ export const PropertyValue: FC<Props> = ({
   };
 
   const getPropertyInput = () => {
+    const commonStyle: CSSProperties = {
+      marginBottom: '0px',
+      minWidth: '250px',
+    };
     switch (propertyType.name) {
       case 'string':
       case 'integer':
+      case 'number': {
+        const inputType = ['integer', 'number'].includes(propertyType.name)
+          ? 'number'
+          : 'text';
```

Comment:
I believe we should use witch case only instead clubbing case and checking again and doing something out of it